xmake/rules/c++/modules/modules_support/clang.lua

```diff
@@ -277,7 +277,8 @@ function generate_dependencies(target, sourcebatch, opt)
                     for _, flag in ipairs(compflags) do
                         if flag == "-m64" or flag == "-g" or flag:startswith("-stdlib") or flag:startswith("-m") or
                            (flag:startswith("-f") and not flag:startswith("-fmodule") and not flag:startswith("-fno-implicit-module-maps")) or
-                           flag:startswith("-D") or flag:startswith("-U") or flag:startswith("-I") or flag:startswith("-isystem") or next_flag then
+                           flag:startswith("-D") or flag:startswith("-U") or flag:startswith("-I") or flag:startswith("-isystem") or next_flag or
+                           flag:startswith("-iframework") then
```

Comment:
`-iframework`?  `-framework`?

`-Ixxx`, `-I xxx`  
`-isystemxxx`, `-isystem xxx` 

are the correct flags

Reply:
-iframework does exists, but i found a better way than enabling them one by one :D